### ISSUE TITLE
Old webpackbin links don't work

### ISSUE THREAD

**Issue by arildwtv** [christianalfoni/webpack-bin#226]
<img width="464" alt="screen shot 2017-03-05 at 13 36 01" src="https://cloud.githubusercontent.com/assets/575067/23587271/bf5d5abe-01a8-11e7-9fff-252c944161da.png">

Seems like the deployment of webpackbin2 broke the access to old bins. When I click "Continue to bin", nothing happens. Will this be fixed soon? I'm doing a course tomorrow and have prepared the exercises in webpackbin 1...

**Comment by MarekSalat** [comment]
I had these two bins prepared for a presentation and now I am not able to access them :(
http://www.webpackbin.com/V1lTBavuf
http://www.webpackbin.com/Vyb0dE6Df

**Comment by christianalfoni** [comment]
@MarekSalat Sorry about that, the release did not go as smooth as expected... go to:

http://webpackbin.herokuapp.com/V1lTBavuf
http://webpackbin.herokuapp.com/Vyb0dE6Df

There are your files which you can copy over to new webpackbin.

Working on fixing the redirect and some other stuff now

**Comment by MarekSalat** [comment]
you are my hero, man :) thank you very much

**Comment by christianalfoni** [comment]
The redirects should also now worked, just deployed new version :)

**Comment by Gbuomprisco** [comment]
The redirects seem to work - though the loading seems stuck. No errors in the console though :) (my bin is https://webpackbin.herokuapp.com/416LDqMdz)

**Comment by christianalfoni** [comment]
Yeah, sandbox does not work currently... but you got the code to copy over :)

Have not had time to fix that yet, but thinking maybe I should not... to rather force people move the code over to the new one :)

**Comment by Gbuomprisco** [comment]
yeah - not a problem :) the biggest hassle is to reinstall the deps - it takes a while ... would be amazing to just paste a package.json, but I suspect performance could be an issue. Thanks btw, love webpackbin!

**Comment by christianalfoni** [comment]
@Gbuomprisco Ah, yeah, that would be nice indeed, though taking babysteps on how deps and packaging works :) Made a lot of improvements on the new one, like identifying all entry points of packages, using a CDN instead of a database etc. Should improve a lot :)

**Comment by diegohaz** [comment]
Hey, @christianalfoni. The official website will be https://webpackbin.herokuapp.com now or is this temporary? Should I update my links?

**Comment by christianalfoni** [comment]
@diegohaz Will run it for about a month :) Would be great if you copy over to new webpackbin. Sorry I could not make the transition automatic, was just insane amounts of work.

**Comment by diegohaz** [comment]
Oh, now I got it. The new version is https://www.webpackbin.com/ and the herokuapp is just a backup. So I need to copy the code from the herokuapp to the new version, right?

**Comment by christianalfoni** [comment]
Ah, yeah, that is correct :)

**Comment by fbarrailla** [comment]
The old version seems to be unavailable now... for example when I try to access this link [https://webpackbin.herokuapp.com/4yityG_mG](https://webpackbin.herokuapp.com/4yityG_mG) I got that page :

<img width="301" alt="nothing" src="https://cloud.githubusercontent.com/assets/11709778/25771387/a77411d6-324f-11e7-9b84-9fe5dea41dd3.png">

**Comment by graingert** [comment]
@fbarrailla I've registered the old webpackbin herokuapp route. All URLs now redirect to this issue.

**Comment by zwm0426** [comment]
All the links are failed and most of them redirect to this page, this makes me feel frustrated and weird.

**Comment by graingert** [comment]
@zwm0426 is there something I could redirect to that would be less frustrating?
